lib/hsa/kernels/rfi_chime_timesum_private.cl

```diff
@@ -70,4 +72,8 @@ rfi_chime_timesum(
     output[1 + address] = tmp.s1;
     output[2 + address] = tmp.s2;
     output[3 + address] = tmp.s3;
+
+    //Output the variance for the selected feed
+    if(gx == feed_index / 4)
+        output_var[gy] = sq_power_across_time[feed_index % 4] / mean[feed_index % 4];
```

Comment:
Sorry I was thinking about something else when I asked for this formula, the formula you should use is: 
```suggestion
        output_var[gy] = sq_power_across_time[feed_index % 4] / (sk_step - 1);
```